libvips/foreign/svgload.c

```diff
@@ -300,16 +307,15 @@ vips_foreign_load_svg_parse( VipsForeignLoadSvg *svg, VipsImage *out )
 	 */
         vips_image_pipelinev( out, VIPS_DEMAND_STYLE_FATSTRIP, NULL );
```

Comment:
We might as well check the return here.

Reply:
I initially added a `return`, but then removed it to stay consistent with the behaviour of all the other loaders. Perhaps I should do a follow-up PR that ensures all calls to `vips_image_pipelinev` pass the status back?

Reply:
Ah I see, you're right, sorry.

Let's leave it without the test (I don't think it can ever fail meaningfully) and do that fixup in master.